docs/tutorials/tokens/erc20.md

```diff
@@ -134,18 +156,34 @@ Other parameters:
 
 ## Transfer tokens
 
-You may transfer tokens within a pool by specifying an amount and a destination understood by the connector (i.e. an Ethereum address). With a token contract deployed by the default token factory, only the owner of a token may transfer it away, but a different contract may define its own permission model.
+You may transfer tokens within a pool by specifying an amount and a destination understood by the connector (i.e. an Ethereum address). With a token contract deployed by the default token factory, only the owner of the tokens or another approved account may transfer their tokens, but a different contract may define its own permission model.
 
+#### Request
 `POST` `http://127.0.0.1:5000/api/v1/namespaces/default/tokens/transfers`
-
 ```json
 {
-  "amount": 1,
-  "to": "0x07eab7731db665caf02bc92c286f51dea81f923f"
+  "amount": "10000000000000000000",
+  "to": "0xa4222a4ae19448d43a338e6586edd5fb2ac398e1"
 }
 ```
 
-> **NOTE:** When transferring a non-fungible token, the amount must always be `1`. The `tokenIndex` field is also required when transferring a non-fungible token.
+#### Response
+```json
+{
+    "type": "transfer",
+    "localId": "61f0a71f-712b-4778-8b37-784fbee52657",
+    "pool": "e1477ed5-7282-48e5-ad9d-1612296bb29d",
+    "connector": "erc20_erc721",
+    "key": "0x14ddd36a0c2f747130915bf5214061b1e4bec74c",
+    "from": "0x14ddd36a0c2f747130915bf5214061b1e4bec74c",
+    "to": "0xa4222a4ae19448d43a338e6586edd5fb2ac398e1",
+    "amount": "10000000000000000000",
+    "tx": {
+        "type": "token_transfer",
+        "id": "c0c316a3-23a9-42f3-89b3-1cfdba6c948d"
+    }
+}
+```
 
 Other parameters:
 - You must specify a `pool` name if you've created more than one pool
```

Comment:
A few lines down, there is a broken link to "key concepts" about messaging.

Reply:
Also, could you remove the trailing `:` on the "Broadcast message" and "Private message" headings below? Looks out of place in the Table of Contents.